scripts/plugins/deploy_plugins.py

```diff
@@ -45,7 +48,8 @@ def deploy_plugins():  # pylint: disable=too-many-locals,too-many-branches,too-m
 
     bitops_dir = "/opt/bitops"
     bitops_deployment_dir = "/opt/bitops_deployment/"
-    bitops_plugins_dir = bitops_dir + "/scripts/plugins/"
+    bitops_plugins_dir = BITOPS_plugin_dir
+    bitops_installed_plugins_dir = BITOPS_installed_plugins_dir
```

Comment:
Why the additional variable assignment here?
```suggestion
```
We could just use the constant where needed